docs/notes/2.23.x.md

```diff
@@ -124,6 +124,10 @@ The default version of the `ruff` tool has been updated from 0.4.4 to 0.4.9.
 
 The default version of the [Pex](https://docs.pex-tool.org/) tool has been updated from 2.3.1 to [2.11.0](https://github.com/pex-tool/pex/releases/tag/v2.11.0).
 
+[A new option `[pex-cli].args`](https://www.pantsbuild.org/2.23/reference/subsystems/pex-cli#args) has been added to be able to pass arbitrary arguments to the `pex` tool as part 
```

Comment:
```suggestion
[A new option `[pex-cli].global_args`](https://www.pantsbuild.org/2.23/reference/subsystems/pex-cli#global_args) has been added to be able to pass arbitrary arguments to the `pex` tool as part 
```

Reply:
Oopsie, this didn't make it to the commit, sorry! Done.